libs/features/mas/web-components/test/merch-card.ccd-slice.test.html.js

```diff
@@ -0,0 +1,54 @@
+// @ts-nocheck
+import { runTests } from '@web/test-runner-mocha';
+import { expect } from '@esm-bundle/chai';
+
+import { mockLana } from './mocks/lana.js';
+import { mockFetch } from './mocks/fetch.js';
+import { mockConfig } from './mocks/config.js';
+
+import '../src/merch-offer.js';
+import '../src/merch-offer-select.js';
+import '../src/merch-quantity-select.js';
+
+import { appendMiloStyles, delay } from './utils.js';
+import { mockIms } from './mocks/ims.js';
+import { withWcs } from './mocks/wcs.js';
+import mas from './mas.js';
+
+const skipTests = sessionStorage.getItem('skipTests');
+
+runTests(async () => {
+    mockIms();
+    mockLana();
+    await mockFetch(withWcs);
+    await mas();
+    appendMiloStyles();
```

Comment:
we should remove appendMiloStyles, since mas is in milo now. Someone needs to remove it all together.

Reply:
Removed.